root/etc/cont-init.d/30-config

```diff
@@ -3,12 +3,5 @@
 
 cd /app/conreq || return
 
-python3 ./manage.py migrate --noinput
-python3 ./manage.py collectstatic --link --noinput
-python3 ./manage.py compress
-s6-setuidgid abc python3 ./manage.py run_huey --quiet &
-
-# permissions
-chown -R abc:abc \
-    /app \
-    /config
+# Permissions
+python3 manage.py preconfig_conreq $(id -u abc) $(id -g abc)
```

Comment:
Why not use the existing envs here?

Reply:
Because I forgot those existed.

Do you want to push a commit to this branch to use PUID and GUID envs?